pyproject.toml

```diff
@@ -17,7 +17,15 @@ classifiers = [
     "Operating System :: Microsoft :: Windows",
     "Operating System :: POSIX",
     "Operating System :: Unix",
-    "Operating System :: MacOS"
+    "Operating System :: MacOS",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
I would rather avoid the whole list in classifiers. It is already well documented here:
https://github.com/codespell-project/codespell/blob/6d001a1b32b53284278f297cce810e3fa453eafb/pyproject.toml#L7

This seems sufficient:
```yaml
    "Programming Language :: Python",
    "Programming Language :: Python :: 3",
    "Programming Language :: Python :: 3 :: Only",
```

Reply:
Thanks for reviewing. I have made the suggested change in the latest commit.

As a consumer of third party packages, I find these classifiers useful. To me, they communicate that the package is verified and tested with all listed version. As well, it is easy for scripts and bots to consume this metadata to track trends across the Python community (e.g. https://hugovk.github.io/drop-python/3.6/). NBD though.

Reply:
It is even easier for scripts and bots to consume `requires-python`, which is the authoritative source of information.

Reply:
Ah, I get it: `requires-python` tells us that codespell is supposed to work with Python >= 3.7, while the classifiers convey the message that codespell has been actually tested with a specific Python version.

Reply:
Another difference to consider, requires-python is a version specifier while the classifiers are on/off tags. The spec for how to interpret and write version specifiers isn't nothing and while a script could do it, I wouldn't describe it as easier.

https://peps.python.org/pep-0440/

Reply:
Thinking about it again, how easy it is to parse classifiers is not relevant because the code is written once, then scripts and bots parse automatically. I believe the real problem is the duplication of information, and that the single source of information should be the one actually enforced by code. Now, I understand information is not entirely duplicated in this case – there are nuances. Plus, this duplication is customary in Python.

I don't have a strong opinion on the subject, I'd happily leave the decision to others. Actually, you could put the classifiers back, and we can ask the maintainers.

As an alternative, it might be easier to split in two PRs:
* Remove all classifiers from this PR, change the title to _Add testing for Python 3.11_, and I will merge it immediately.
* Create a new PR with title _Document tested versions of Python_ or something like that, and choose your arguments wisely.